### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Rishikesh233** [ModECI/MDF#407] [General]
Hello everyone, I'm new here. I'm so excited to start contributing. @pgleeson and Ankur Sinha. Sir, I have some doubts about the task.

**Comment by 1awesomeJ** [comment]
Hello everyone, Josh here from Lagos.

**Comment by Br-Al** [comment]
Hello, my name is Kenfack Anafack Alex Bruno, or just Bruno for short. I'm a computer vision learner and part of the Outreachy program. I'm a big fan of everything concerning AI. I'm happy to have the opportunity to contribute to this project. 🫠
